Resources/Prototypes/Reagents/gases.yml

```
@@ -181,6 +198,26 @@
       - !type:ChemVomit
         probability: 0.12
         conditions:
+        - !type:OrganType
+          type: Human
+        - !type:OrganType
+          type: Animal
+        - !type:OrganType
+          type: Vox
```

this doesn't work, it checks for all conditions. you need to do an inverted organtype

```suggestion
        - !type:OrganType
          type: Rat
          shouldHave: false
```